src/components/UserList/UserList.js

```diff
@@ -0,0 +1,79 @@
+import React, { Component } from 'react';
+import {
+ StyleSheet,
+ Text,
+ View,
+ Navigator,
+ Image,
+ ListView,
+} from 'react-native';
+import Header from './Header';
+import Footer from './Footer';
+import Row from './Row';
+
+const API_KEY = '73b19491b83909c7e07016f4bb4644f9:2:60667290';
+const QUERY_TYPE = 'hardcover-fiction';
+const API_STEM = 'http://api.nytimes.com/svc/books/v3/lists';
+const ENDPOINT = `${API_STEM}/${QUERY_TYPE}?response-format=json&api-key=${API_KEY}`;
+
+class UserList extends Component {
+  constructor(props) {
+    super(props);
+    let dataSource = new ListView.DataSource({
+      rowHasChanged: (row1, row2) => row1 != row2,
+    });
+    this.state = {
+      dataSource: dataSource.cloneWithRows([1, 2, 3, 4, 5]),
```

Comment:
What is this [1,2,3,4,5]? Is this duplicating the mock Row object above? More comments please!